pkg/syscall/syscall.go

```diff
@@ -31,6 +31,11 @@ func SetupShutdownNotify(sigCh chan os.Signal) {
 // CreateProcessGroupID creates a process group ID for the current process.
 // Reference - https://man7.org/linux/man-pages/man2/setpgid.2.html.
 func CreateProcessGroupID() {
+	// Below is some excerpt from the above link Setpgid() -
+	// setpgid() sets the PGID of the process specified by pid to pgid.
+	// If pid is zero, then the process ID of the calling process is
+	// used.  If pgid is zero, then the PGID of the process specified by
+	// pid is made the same as its process ID.
```

Comment:
This is from the manpage, but rather it should be in context of the Dapr CLI.

Reply:
just thinking what to put because putting in context of dapr run won't be right as it is now kind of utility can be extended or modified based on further usecases.

we have put some information wrt `dapr run -f` from where this gets invoked.